Examples/WindowsFormsApplication/readme.md

```diff
@@ -0,0 +1,127 @@
+# Windows Forms sample
+
+This sample demonstrates using SSO through AAD in a Windows Forms application.
+In order to use this sample you will have to perform a number of steps.
+
+## Setup
+
+To support SSO a new application needs to be added to the tenant that will be used to authenticate the user.
+
+### _Login to the [Azure portal](https://portal.azure.com) and go to `App registrations`_
+
+![App registrations](./images/AppRegistrations.png)
+
+### _Register a new application_
+
+![New app registration](./images/NewRegistration.png)
+![Register an application](./images/RegisterApplication.png)
+
+### _Take note of the appropriate settings_  
+
+If you now look at the application overview a number of details are shown that will be used later.
+Store the values of the application ID and the tenant ID.
+
+![Initial app details](./images/InitialAppDetails.png)
+
+### _Add a platform for our Windows Forms application_
+
+In order for our app to use this application registration you will have to tell AAD through which means it will authenticate.
+
+![Add a platform](./images/AddPlatform.png)
+
+### _Add support for `Mobile and desktop applications`_
+![Mobile and desktop applications](./images/AddMobileDesktopApplication.png)
+
+### _Setup `Redirect URIs`_
+
+In order to support the ability to show the browser to select the account to use for logging in to Nfield the following URIs should be configured. The `http://localhost` URI is needed when using .Net Core, it will allow the application to capture the result of the login from the browser.
+
+![Redirect URIs](./images/RedirectUris.png)
```

Comment:
do we also need the /common/nativeclient thingy?

Reply:
If you use regular .Net you can use that one, for .Net Core (and up) this is needed as there they don't support the embedded browser flow.